### DISCUSSION TITLE
Preview Access - Extension Issue

### DISCUSSION THREAD

**Discussion by usrnatc** [community/community#10978] [Copilot Conversations]
I was notified that I was able to start testing the copilot AI but when I install the extension and sign in, I am greeted by this message:
![image](https://user-images.githubusercontent.com/79397859/152464732-3efd2cdf-88a2-4a33-8a52-581b5b7a7ab9.png)

I am Currently using the latest version of VS Code (stable) on Windows 10 x64 21H1 OS Build 19043.1110

Please move to Issues Feedback if necessary, :^)

**Comment by davecheney** [comment]
I have confirmed that your invite is active. If you have multiple github accounts, please make sure that vscode is signed in as the user @[usrnatc](https://github.com/usrnatc)

<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
